test/langtools/jdk/javadoc/doclet/testJavaFX/TestJavaFX.java

```diff
@@ -419,6 +419,7 @@ public final BooleanProperty somethingProperty() {
         javadoc("-d", "out5",
                 "--javafx",
                 "--disable-javafx-strict-checks",
+                "--no-platform-links",
```

Comment:
It just happens so that yesterday I [asked](https://github.com/openjdk/jdk/pull/4051#pullrequestreview-660862882) Hannes about proliferation of `--no-platform-links` in tests and now we have 3 more of those in this PR alone.

Reply:
I was getting more messages reporting that the classes could not be found.  It was throwing off warning counts in golden output. My unproven suspicion is that we may not have been accurately counting warnings before this change.